auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/external/DeltaUserRegistrationController.kt

```diff
@@ -48,6 +48,15 @@ class DeltaUserRegistrationController(
     }
 
     private suspend fun registerGet(call: ApplicationCall) {
+        val ssoEmail = call.parameters["sso_email"]
```

Comment:
Could this variable be named something else, I'm not sure quite what best in my head is "notRequiredSSOEmail" which is rather gross but I think just ssoEmail is a little confusing to read? Maybe "fromSSOEmail"?